### ISSUE TITLE
Cache descriptions showing blank

### ISSUE THREAD

**Issue by OgreB** [cgeo/cgeo#7291]
##### Detailed steps causing the problem:
- open non saved cache.
- goto description.
- no description show. Hint will show

Cgeo 2018.10.13

##### Is the problem reproducible for you?
Yes

##### System information:
```--- System information ---
Device: Pixel XL (marlin, google)
Android version: 9
Android build: PQ2A.190205.003
c:geo version: 2018.10.13
Google Play services: enabled - 15.0.90 (100400-231259764)
Low power mode: inactive
Compass capabilities: yes
Rotation vector sensor: present
Orientation sensor: present
Magnetometer & Accelerometer sensor: present
Direction sensor used: rotation vector
Hide own/found: false
Map strategy: auto
HW acceleration: enabled (default state)
System language: en_US
System date format: M/d/yy
Debug mode active: no
System internal c:geo dir: /data/user/0/cgeo.geocaching (56.5 GB free) internal
User storage c:geo dir: /storage/emulated/0/cgeo (56.5 GB free) external non-removable
Geocache data: /storage/emulated/0/Android/data/cgeo.geocaching/files/GeocacheData (56.5 GB free) external non-removable
Database: /storage/emulated/0/Android/data/cgeo.geocaching/files/databases/data (79.0 MB) on user storage
Fine location permission: granted
Write external storage permission: granted
Geocaching sites enabled:
   geocaching.com: Logged in (Login OK) / PREMIUM
Geocaching.com date format: MM/dd/yyyy
Installed c:geo plugins:  none
--- End of system information ---
ttach system information here if available (see c:geo menu -> About c:geo -> Swipe right to System)
Keep the apostrophe at beginning and end to have it properly formatted
```

##### Other comments and remarks:
e.g. Reference to other issues, projects, sources, etc.

**Comment by Lineflyer** [comment]
Can reproduce.Looks like a website change, which would mean we could be forced to release rather soon. :(Von meinem Smartphone gesendet.
null

**Comment by rsudev** [comment]
@Lars: With nightly or release version? I checked quickly and didn't see an obvious problem. Sample cache?

**Comment by RolandRosenfeld** [comment]
@rsudev Just tried with 2019.02.16-NB-e58643 and have the same issue as with the release version.
I can reproduce the issue with every cache, see for example GC7RVWZ, there is only the hint but no cache description.
My wife told me, that she sees the short description on some caches, which have one defined, but this is the minority of caches, and the full description is missing there, too.

**Comment by Lineflyer** [comment]
@rsudev Affects all caches for me.
As @RolandRosenfeld wrote: Long description is empty. Other description fields look normal.

If someone can confirm I suggest to raise  the default "website change" notification to inform users.

**Comment by smartass101** [comment]
> see for example GC7RVWZ, there is only the hint but no cache description.

I can confirm this with a clean install on a separate Android phone. Downloaded this example cache by code.
It's a shame this has to happen when the weather is so nice today here ;)

**Comment by alan666notb** [comment]
I can confirm we see this on unpublished caches as well.
(Make changes to text, save on website, refresh cgeo, changes aren't reflected in cgeo).
Changing other things such as placed date do get updated.

**Comment by bekuno** [comment]
@Lineflyer I can confirm it also. The website change notification should be raised.

**Comment by Lineflyer** [comment]
Notification raisedVon meinem Smartphone gesendet.
null

**Comment by Lineflyer** [comment]
I took a look into the debug log, but it shows nothing with respect to this problem.
I guess the website content changed in a way we fail to identify the description section and thus not parsing it.

Is someone already looking into this?

**Comment by bekuno** [comment]
Please see at PR #7295

**Comment by Lineflyer** [comment]
@bekuno Cool, thanks!

BTW, I posted to social networks:
```
At the moment there is the problem, that cache descriptions are not shown for some users.

We are working on it, however it might take some more time as usual to fix this problem.
The reason is, that Google recently changed its requirements for all apps on Google Play and we were therefore forced to do some major changes in c:geo, which are still not completed. Thus we are not really ready to release a new stable version...we will see what we can do to help our users as soon as possible.
```

**Comment by Lineflyer** [comment]
Fix tested and working in 2019.02.17-NB1
